internal/events/get.go

```diff
@@ -47,15 +58,12 @@ func get(
 	_ command.GlobalFlags,
 	services *services.Services,
 ) (command.Result, error) {
-	name := args[0]
-	start := args[1] // block height range start
-	end := ""        // block height range end
-
-	if len(args) == 3 {
-		end = args[2]
+	start, end, err := services.Events.CalculateStartEnd(eventsFlags.Start, eventsFlags.End, eventsFlags.Last)
+	if err != nil {
+		return nil, err
 	}
 
-	events, err := services.Events.Get(name, start, end)
+	events, err := services.Events.Get(args, start, end, eventsFlags.Batch, eventsFlags.Workers)
```

Comment:
As in the comment above, we should have fine-grained API which defines each argument separately.